src/components-examples/angular/sidebar/collapsible-sidebar/collapsible-sidebar-example.html

```diff
@@ -0,0 +1,60 @@
+<div class="sidebar-frame">
+  <sbb-icon-sidebar-container>
+    <sbb-icon-sidebar [position]="position.value">
+      <a sbbIconSidebarItem label="Station" href="">
+        <sbb-icon svgIcon="station-small"></sbb-icon>
+      </a>
+      <a sbbIconSidebarItem label="Lockers" class="sbb-active" (click)="sidebar.toggle()">
```

Comment:
@mcilurzo, @kyubisation, @jeripeierSBB this needs to be discussed.

The items of the `SbbIconSidebar` are links (`<a />` tags). The links are usually used to navigate to new pages having it's own `SbbSidebar`. In the demo I am (mis)using the active link to toggle the sidebar collapsed state.

I think that's not an optimal solution from UX and accessibility perspective. What do you think?

Reply:
As discussed, I agree, but do not have a better solution.
An alternative might be to have a burger button in the content, but I'm not sure whether that works well.
@mcilurzo ?